plugin-api/src/main/java/org/hyperledger/besu/plugin/services/BesuEvents.java

```diff
@@ -109,9 +109,9 @@
      * <p>The block may not have been imported to the local chain yet and may fail later
      * validations.
      *
-     * @param blockHeader the new block header.
+     * @param blockPropagated the new block header.
```

Comment:
The comment needs updating too, as now it is no longer simple the block header.
```suggestion
     * @param blockPropagated block being propagated.
```